configs/openSUSE/dbus-services.toml

```diff
@@ -214,6 +214,20 @@ nodigests = [
     "/usr/share/dbus-1/system.d/org.kde.ksysguard.processlisthelper.conf",
 ]
 
+[[FileDigestGroup]]
+package  = "libksysguard6-plugins"
+type     = "dbus"
+note     = "send signals to and change scheduling properties of arbitrary processes"
+bug      = "bsc#1217182"
+[[FileDigestGroup.digests]]
+path     = "/usr/share/dbus-1/system.d/org.kde.ksysguard.processlisthelper.conf"
+digester = "xml"
+hash     = "e08be1fffd5b7ebddb31f396f603e7a9fac08784c5473f37195262b55c663d45"
+[[FileDigestGroup.digests]]
+path     = "/usr/share/dbus-1/system-services/org.kde.ksysguard.processlisthelper.service"
+digester = "shell"
+hash     = "dc9ea7f2e2c649aa9bd30ddfd3e497d23de6cf801424460723a0df0a56514aa"
```

Comment:
It seems that you dropped a leading "a" from the digest. OBS has the digest as:

```
adc9ea7f2e2c649aa9bd30ddfd3e497d23de6cf801424460723a0df0a56514aa
```

By the way: is it right that hashes are 65 characters long?

Reply:
The hexdigests should be 64 characters, where do you see it differently?

I'll fix this missing `a`. We could add a unit test that verifies that hashes always have the proper length. This seems to be a common error.

Reply:
I rebased the corresponding commit.

Reply:
> The hexdigests should be 64 characters, where do you see it differently?

I'm dumb, I checked it with `echo "..." | wc -c` instead of `echo -n "..." | wc -c`, so I counted the newline as well. I knew it didn't make sense that sha256 produces a 65 character output.

Reply:
> I'll fix this missing a. We could add a unit test that verifies that hashes always have the proper length. This seems to be a common error.

Yes that was one of the main issues that I wanted to address in CI. However now with the `buildlog2whitelist` helper script, hopefully such instances should just disappear. I assume you produced this entry manually or before the script was in use?